releasenotes/notes/profile-upload-2.4-8277b229fda6ff53.yaml

```
@@ -0,0 +1,3 @@
+other:
```

```suggestion
upgrade:
```
If I'm not mistaken, this is an upgrade to the profiler since we're supporting a new backend endpoint and format.